### PR TITLE
#5241 Rewrite TestShabondi#testCheckBrokerDependencies and TestShabon…

### PR DESCRIPTION
**Pull request by chia7712**
…di#testCheckTopicDependencies by unit test

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5241)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/TopicRoute.scala [hunk 1]
```diff
@@ -223,6 +223,7 @@ private[configurator] object TopicRoute {
         .topic(TopicKey.of(key.group(), key.name()), DataCondition.STOPPED)
         .allConnectors()
         .allStreams()
+        .allShabondis()
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/TopicRoute.scala:226]: 這是個嚴重的bug (1/2)

File: ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/BrokerRoute.scala [hunk 2]
```diff
@@ -183,10 +183,10 @@ object BrokerRoute {
           report =>
             (
               report.brokerClusterInfos.head._1,
-              report.runningWorkers,
-              report.runningStreams,
-              report.runningShabondis,
-              report.runningTopics
+              report.workerClusterInfos.keys.toSeq,
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/BrokerRoute.scala:186]: 這是個嚴重的bug (2/2)

**Comment by chia7712** [comment]
很不幸的發現我們在route這邊漏掉蠻多檢查，真是傷心

File: ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/TopicRoute.scala (hunk 1 above)
Comment (jackyoh) [reply ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/TopicRoute.scala:226]: 這裡是指檢查 shabondis cluster info 嗎？

Comment (vitojeng) [reply ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/TopicRoute.scala:226]: > 這是個嚴重的bug (1/2)

It's my bad. Sorry.

**Pull request merged**